R/extract.data.R

```diff
@@ -12,6 +12,8 @@ NULL
   data.list <- response.content[['data']]
   if (is.null(data.list)) {
     data.list <- list()
+  } else {
+    print(paste("response.content[['data']] is", data.list))
```

Comment:
Why do we need to print this? If we actually need to give information to the user then we should use the `message` function